tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -52,3 +68,48 @@ def dismiss_tooltip(self):
         confirm_element = Wait(self.marionette).until(expected.element_present(*self._button_popup_intro))
         Wait(self.marionette).until(expected.element_displayed(confirm_element))
         confirm_element.tap()
+
+
+class BugzillaLiteStage(BugzillaLite):
+
+
+    name =  'Bugzilla Lite Stage'
+    manifest_url = 'http://bzlite-staging.herokuapp.com/manifest.webapp'
+
+    def dismiss_content(self):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._content_locator))))
+        self.marionette.find_element(*self._close_locator).tap()
+        Wait(self.marionette).until(expected.element_displayed(*self._login_form_locator))
+        import time
+        time.sleep(1)
+
+    def create_new_bug(self, title, description):
+        self.marionette.find_element(*self._create_button).tap()
+        from gaiatest.apps.bzlite.regions.create_bug import CreateBug
+        
+        CreateBug(self.marionette)._fill_title(title)
+        CreateBug(self.marionette)._fill_description(description)
+        CreateBug(self.marionette)._fill_picture()
+        CreateBug(self.marionette)._submit()
+
+        time.sleep(10)
+        Wait(self.marionette,timeout=20).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._bugpage_locator))))
```

Comment:
Perhaps add interval=1 to this Wait, since this one is probably take some time anyway?